aioredisqueue/task.py

```diff
@@ -16,8 +16,11 @@ def generate_id(payload=None):
         return str(uuid.uuid4())
 
     @staticmethod
-    def format_payload(payload):
-        return payload
+    def format_payload(payload, serializer='raw'):
+        if serializer == 'raw':
+            return payload
+        else:
+            raise NotImplementedError
```

Comment:
Ideally it would be great to have it the way kombu implements it:
https://kombu.readthedocs.io/en/stable/userguide/serialization.html#creating-extensions-using-setuptools-entry-points

but at least we can just create a `serialization` method with `get_serializer()` function which is basically some `dict.__getitem__`